src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -76,8 +77,147 @@ def __init__(self, state):
     def predict(self, dataset):
         return self.__proxy__['detector'].predict(dataset)
 
-    def evaluate(self, dataset, metric="auto"):
-        return self.__proxy__['detector'].evaluate(dataset, metric)
+    def evaluate(self, dataset, annotations, metric="auto", output_type='dict', iou_threshold=None, 
```

Comment:
Use kwargs here.

Reply:
Using it in all three export_coreml, evaluate and predict. Is that alright?